code/game/jobs/job/captain.dm

```diff
@@ -17,7 +17,6 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 	playtimerequired = 2500
 	wage = WAGE_COMMAND
 
-	health_modifier = 1
```

Comment:
colonist+ should have 1 more hp imo

Reply:
honestly i really dont see why

Reply:
the premier already has /things/, why give them a /singular/ hp

Reply:
Its colonist+ they should have something to denote that they are better then the rest

Reply:
ඞ